libpyvinyl/BaseFormat.py

```diff
@@ -0,0 +1,250 @@
+from abc import abstractmethod
+from libpyvinyl.AbstractBaseClass import AbstractBaseClass
+from libpyvinyl.BaseData import BaseData
+import numpy as np
+
+
+class BaseFormat(AbstractBaseClass):
+    """The abstract format class. It's the interface of a certain data format."""
+    def __init__(self):
+        # This class is not designed in a object-oriented model, rather a collection of functions.
```

Comment:
Yes, I borrowed the idea of inheriting from the OO concepts. The comment was intended to explain why the `__init__` is empty. I've removed it since it's a bit confusing.